src/main.js

```diff
@@ -24,27 +46,44 @@ app.listen(3000, () => {
 let errorIt = (res, err) => res.json({error: err})
 
 app.get('/', (req, res) => {
-  let routes = app._router.stack
-    .filter(val => val.route)
-    .map(val => `${val.route.stack[0].method.toUpperCase()} ${val.route.path}`)
+  res.status(200).json(
+    app._router.stack
+      .filter(val => val.route)
+      .map(val => `${val.route.stack[0].method.toUpperCase()} ${val.route.path}`)
+  )
+})
 
-  res.json(200, routes)
+app.get('/users', (req, res) => {
```

Comment:
WTF?
It returns the list of all users in-memory.
It allows us to use different accounts for different user!